src/block-components/typography/edit.js

```diff
@@ -0,0 +1,265 @@
+						/>
+						<AdvancedSelectControl
+							label={ __( 'Transform', i18n ) }
+							options={ [
+								{ label: __( 'Default', i18n ), value: '' },
+								{ label: __( 'Uppercase', i18n ), value: 'uppercase' },
+								{ label: __( 'Lowercase', i18n ), value: 'lowercase' },
+								{ label: __( 'Capitalize', i18n ), value: 'capitalize' },
+								{ label: __( 'None', i18n ), value: 'none' },
+							] }
+							attribute="textTransform"
+							hover="all"
+						/>
+						<AdvancedRangeControl
+							label={ __( 'Line-Height', i18n ) }
+							attribute="lineHeight"
+							units={ [ 'px', 'em' ] }
+							min={ [ 1, 0.1 ] }
+							max={ [ 100, 10 ] }
+							step={ [ 1, 0.1 ] }
+							placeholder={ [ 30, 1.5 ] }
+							allowReset={ true }
+							initialPosition={ [ 37, 1.8 ] }
+							responsive="all"
+							hover="all"
+						/>
+						<AdvancedRangeControl
+							label={ __( 'Letter Spacing', i18n ) }
+							attribute="letterSpacing"
+							min={ -5 }
+							max={ 10 }
+							step={ 0.1 }
+							allowReset={ true }
+							placeholder="0"
+							hover="all"
+						/>
+					</ButtonIconPopoverControl>
+					<AdvancedRangeControl
+						label={ __( 'Size', i18n ) }
+						allowReset={ true }
+						attribute="fontSize"
+						units={ [ 'px', 'em' ] }
+						min={ [ 0, 0 ] }
+						max={ [ 150, 7 ] }
+						step={ [ 1, 0.05 ] }
+						placeholder="32"
+						responsive="all"
+						hover="all"
+					/>
+					<AdvancedToolbarControl
+						controls={ [
+							{
+								value: '',
+								title: __( 'Single', i18n ),
+							},
+							{
+								value: 'gradient',
+								title: __( 'Gradient', i18n ),
+							},
+						] }
+						isSmall={ true }
+						fullwidth={ false }
+						attribute="textColorType"
+						hover="all"
+						onReset={ () => {
+							updateAttributes( {
+								[ getAttributeName( 'textColor1', 'desktop', state ) ]: '',
+								[ getAttributeName( 'textColor2', 'desktop', state ) ]: '',
+							} )
+						} }
+					/>
+					<ColorPaletteControl
+						label={ getAttribute( 'textColorType', 'desktop', state ) === 'gradient' ? sprintf( __( '%s Color #%s', i18n ), label, 1 )
+							: sprintf( __( '%s Color', i18n ), label ) }
+						attribute="textColor1"
+						hover="all"
+					/>
+					{ getAttribute( 'textColorType', 'desktop', state ) === 'gradient' && (
+						<Fragment>
+							<ColorPaletteControl
+								label={ sprintf( __( '%s Color #2', i18n ), label ) }
+								attribute="textColor2"
+								hover="all"
+							/>
+
+							<AdvancedRangeControl
+								label={ __( 'Gradient Direction (degrees)', i18n ) }
+								attribute="textGradientDirection"
+								hover="all"
+								min={ 0 }
+								max={ 360 }
+								step={ 10 }
+								allowReset={ true }
+							/>
+						</Fragment>
+					) }
+					{ ! disableAlign && (
+						<AlignButtonsControl
+							label={ __( 'Align', i18n ) }
+							attribute="textAlign"
+							responsive="all"
+						/>
+					) }
```

Comment:
Let's remove this. If there's alignment, the block should just use the `Alignment` block component